lang/java/reef-wake/wake/src/test/java/org/apache/reef/wake/test/avro/ProtocolSerializerTest.java

```diff
@@ -126,14 +126,13 @@ private RemoteManager getTestRemoteManager(final String identifier) {
         localAddressProvider, tcpPortProvider);
 
     } catch (final Exception e) {
```

Comment:
Also, why do we catch (and suppress) exceptions here? This method is used for tests only, so let exceptions fly and the unit tests fail if something goes wrong! I would remove the entire `try` / `catch` block here.